library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c

```diff
@@ -1805,7 +1806,7 @@ common_test_case_t m_spdm_test_group_measurements[] = {
     {SPDM_RESPONDER_TEST_CASE_MEASUREMENTS_INVALID_REQUEST,
      "spdm_test_case_measurements_invalid_request",
      spdm_test_case_measurements_invalid_request,
-     spdm_test_case_measurements_setup_version_any,
+     spdm_test_case_measurements_setup_version_11,
```

Comment:
should it also applicable to version 1.2 ?

Reply:
Yes. The invalid request is also applicable to version 1.2. I have made it back to `spdm_test_case_measurements_setup_version_any`.